packages/server/graphql/mutations/resetRetroMeetingToGroupStage.ts

```diff
@@ -30,9 +46,12 @@ const resetMeetingToStage = {
 
     // AUTH
     const viewerId = getUserId(authToken)
-    const meeting = await dataLoader.get('newMeetings').load(meetingId)
+    const meeting = (await dataLoader.get('newMeetings').load(meetingId)) as MeetingRetrospective
     if (!meeting) return standardError(new Error('Meeting not found'), {userId: viewerId})
-    const {createdBy, facilitatorUserId, phases, teamId, meetingType, meetingCount} = meeting
+    const {createdBy, facilitatorUserId, phases, meetingType} = meeting
+    if (meetingType != 'retrospective') {
```

Comment:
`stageId` is no longer necessary since this mutation will only reset to the group stage